tests/notifier/notifiers/test_data_attachment.csv

```diff
@@ -0,0 +1,3 @@
+resource_id,resource_type,full_name,rule_index,rule_name,violation_type,violation_data

```

Comment:
nit: We have the convention of putting test data like this, into a `test_data` directory.  So, can you please create `/tests/notifier/notifiers/test_data` package, and move this there?  You will have create an `__init__.py` file too.  And then call this file `expected_attachment.csv`?

Reply:
Done.